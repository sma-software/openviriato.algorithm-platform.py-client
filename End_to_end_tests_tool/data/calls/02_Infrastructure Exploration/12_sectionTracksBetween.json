{
    "restResource": "/section-tracks",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromNodeID",
            "value": "161",
            "optional": false
        },
        {
            "key": "ToNodeID",
            "value": "348",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetSectionTracksBetween(fromNodeID, toNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64" ],
    "subcategory": "Get Section Tracks Between"
}
