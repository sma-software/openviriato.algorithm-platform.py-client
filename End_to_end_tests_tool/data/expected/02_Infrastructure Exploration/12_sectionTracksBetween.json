[{"ID":885,"Code":"838","SectionCode":"61010","Weight":37040,"DebugString":"sectiontrack:s_61010 n_85AR 838"},{"ID":886,"Code":"2","SectionCode":"61010","Weight":37040,"DebugString":"sectiontrack:s_61010 n_85AR 2"}]